content/github/authenticating-to-github/working-with-ssh-key-passphrases.md

```diff
@@ -18,10 +18,10 @@ With SSH keys, if someone gains access to your computer, they also gain access t
 You can change the passphrase for an existing private key without regenerating the keypair by typing the following command:
 
 ```shell
-$ ssh-keygen -p
+$ ssh-keygen -p -f ~/.ssh/id_ed25519
 # Start the SSH key creation process
```

Comment:
When you enter the above command the output is the `Key has comment ...` line, so we can delete this and the next line:
```suggestion
```